variants/aws-dev/Cargo.toml

```diff
@@ -28,6 +27,7 @@ included-packages = [
 # core
     "release",
     "kernel-6.1",
+    
```

Comment:
nit: I'd suggest pruning (or standardizing) the added whitespace, since it's not consistent between files.